packages/redux/src/exchanges/selectors.ts

```diff
@@ -0,0 +1,99 @@
+import {
+  getError,
+  getExchangeBookRequest as getExchangeBookRequestGetter,
+  getExchangeFilters as getExchangeFiltersGetter,
+  getIsLoading,
+  getResult as Result,
```

Comment:
This rename is not necessary, you can keep using `getResult` as it does not clash here.

Reply:
Fixed 👍